sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc

```diff
@@ -813,6 +813,16 @@ public:
       const ext::oneapi::experimental::image_mem_handle Src,
       ext::oneapi::experimental::image_mem_handle Dest,
       const ext::oneapi::experimental::image_descriptor &ImageDesc);
+
+  // Device to device copy with offsets and extent
+  void ext_oneapi_copy(
```

Comment:
are we sure about this name?  For something so generic like  `ext_oneapi_copy` I would expect something that handles any type of memory (not just image data) and maybe across all address spaces (host, device). Much like queue.copy() is now.

But, the comments on this routine suggest its capability is much more focused. If so, I think it's name should express that.  maybe something like `ext_oneapi_d2d_img_copy`   or `ext_d2d_img_copy`   ?

Reply:
There are still new overloads planned to be added. Such as host to host and device usm to device usm.

Reply:
I have been thinking that perhaps the amount of `ext_oneapi_copy` overloads we have is getting a bit unwieldy. Especially considering a number additional ones need to be added to support more device to device copy variants. 

I don't think it would be a good idea to start making changes involving other copy functions here. But something to be looked into later.

Reply:
sorry for not responding to your comments earlier. If there are other overloads that handle other data types, it wouldn't hurt to mention that in the comment.  Even just "this overload of ext_copy is for bindless images".

Reply:
No problem, thanks for the approval! On your suggestion, it would likely be best to apply a comment like you suggest to all the overloads at the same time which is a bit out of scope it feels for this PR. As there are 50~ of them. queue.hpp and handler.hpp need a bit of a refactor which is very tentatively planned early next year. Double checking the order of the functions and their corresponding comments. Comments stating they are overloads for bindless images can be added then.